app/styles/layouts/editor.css

```diff
@@ -251,7 +251,19 @@
     line-height: 0;
 }
 
-
 .gh-editor-header-small .post-settings {
     padding: 13px 15px;
 }
+
+.gh-editor-wordcount {
+     position: fixed;
+     bottom: 0px;
+     padding:10px;
+ }
+
+ @media (max-width: 1200px) {
```

Comment:
The resize is in JS so it's heavily debounced - it doesn't look like you're giving it time to settle?